weblab_cg/templates/chaste/normal_model.cpp

```diff
@@ -0,0 +1,157 @@
+//! @file
+//! 
+//! This source file was generated from CellML.
+//! 
+//! Model: {{model_name}}
+//! 
+//! Processed by webalab_cg - CellML Tools in Python: https://github.com/ModellingWebLab/weblab-cg
+//!     (translators: , webalab_cg: , options: normal)
```

Comment:
Get rid of this. Pycml used it to stick the git revision in generated code; if we're not doing this it's pointless.

Reply:
Adding the git revision is a nice idea

Reply:
Quite nice to know, for posterity, what commits of weblab_cg / cellmlmanip were involved I suppose

Reply:
Actually looking at PyCml code it doesn't have git revision on it.